README.md

```diff
@@ -1 +1 @@
-# accula
\ No newline at end of file
+# Advanced Code Clones UniversaL Analyzer
```

Comment:
Not issue-related change

Reply:
I think it doesn't matter in this case if you aren't going to create a separate task to update README